ibis-server/docs/development.md

```diff
@@ -48,6 +48,17 @@ To run the tests:
 - Build the image: `just docker-build`
 - Run the container: `just docker-run`
 
+### Run the testing
+- Preapre the Wren Engine server (see [Wren Engine Example](../example/README.md)
+- Run pytest with `WREN_ENGINE_ENDPOINT` env. (The default port of Wren Engine is 8080)
+  - run all test
+  ```
+  WREN_ENGINE_ENDPOINT=http://localhost:8080 just test
+  ```
+  - run specific test using [pytest args](https://docs.pytest.org/en/latest/example/markers.html#using-k-expr-to-select-tests-based-on-their-name)
+  ```
+  WREN_ENGINE_ENDPOINT=http://localhost:8080 just test -m 'postgres'
+  ```
```

Comment:
Could you move the `To run the tests` at L#40 into this block?

Reply:
I guess we can remove `To run the tests` because it's duplicated with this block.

Reply:
Sure